File: prograimg_python_basic/Exercises/conditional_statements/conditional_statements_-_exercise/lunch_break.py
import math

serial = str(input())
need_time = int(input())
break_time = int(input())

eating = break_time / 8
chill = break_time / 4

left_time = break_time - eating - chill

if need_time <= left_time:
    left = left_time - need_time
    print(f"You have enough time to watch {serial} and left with {math.ceil(left)} minutes free time.")

elif need_time > left_time:
    need = need_time - left_time
    print(f"You don't have enough time to watch {serial}, you need {math.ceil(need)} more minutes.")

File: JS/JavaScript/JS-Basics/ConditionalStatementsExercise/LunchBreak.js
function lunch(inputs) {
    let serial = String(inputs[0]);
    let time = Number(inputs[1]);
    let brake = Number(inputs[2])

    let leftTime = brake - brake / 8 - brake / 4;

    if (leftTime >= time) {
        console.log(`You have enough time to watch ${serial} and left with ${Math.ceil(leftTime - time)} minutes free time.`);
    }
    else {
        console.log(`You don't have enough time to watch ${serial}, you need ${Math.ceil(time - leftTime)} more minutes.`);
    }
}

lunch(["Teen Wolf",
"48",
"60"])

